src/PostgREST/Config.hs

```diff
@@ -172,6 +174,7 @@ readOptions = do
         |## (use "@filename" to load from separate file)
         |# jwt-secret = "foo"
         |# secret-is-base64 = false
+        |# jwt-aud = "yourkeycloakrealm"
```

Comment:
Oh I almost forgot about changing this output, good idea. Maybe we should use a less keycloak-specific example as a string literal?

Reply:
Good call, I'll just refer to audience.